pkg/services/profiles/update.go

```diff
@@ -0,0 +1,146 @@
+package profiles
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/weaveworks/weave-gitops/pkg/git"
+	"github.com/weaveworks/weave-gitops/pkg/gitproviders"
+	"github.com/weaveworks/weave-gitops/pkg/helm"
+	"github.com/weaveworks/weave-gitops/pkg/models"
+
+	"github.com/fluxcd/go-git-providers/gitprovider"
+	helmv2beta1 "github.com/fluxcd/helm-controller/api/v2beta1"
+	"github.com/google/uuid"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const UpdateCommitMessage = "Update Profile manifests"
+
+type UpdateOptions struct {
+	Name         string
+	Cluster      string
+	ConfigRepo   string
+	Version      string
+	ProfilesPort string
+	Namespace    string
+	Kubeconfig   string
+	AutoMerge    bool
+}
+
+// Update updates an installed profile
```

Comment:
It's not perfectly apparent that the fact that there is a HelmRelease IS the way to get that it is installed. Shall we do something like ` checkIfInstalled` at the begin of the function and fail early? So it doesn't have to download and check anything before it even begun that.